target/linux/ath79/dts/ar9344_dlink_dir-835-a1.dts

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "ar9344_dlink_dir-8x5.dtsi"
+
+/ {
+	model = "D-LINK DIR-835 A1";
+	compatible = "dlink,dir-835-a1", "qca,ar9344";
+
+	aliases {
+		led-boot = &orange_power;
+		led-failsafe = &orange_power;
+		led-running = &green_power;
+		led-upgrade = &orange_power;
+		serial0 = &uart;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		green_power: green_power {
+			label = "d-link:green:power";
+			gpios = <&gpio 22 GPIO_ACTIVE_LOW>;
+		};
+
+		/* In /etc/board.d/03_gpio_switches GPIO 20 is turned
```

Comment:
dts files should not contain owrt specific info. how about adding the subnode but simply commenting it out ?

Reply:
That actually makes complete sense, I had not thought about this. I removed the comment and added the wan led define. It doesn't hurt to be there, no need to comment it out. Thanks!